components/motor/gpio/motor_encoder_test.go

```diff
@@ -33,6 +35,7 @@ func (f *fakeDirectionAware) DirectionMoving() int64 {
 }
 
 func TestMotorEncoder1(t *testing.T) {
+	t.Skip()
```

Comment:
@penguinland skips here will unblock you for Digital interrupts.

Reply:
My workaround to get https://github.com/viamrobotics/rdk/pull/2236 to pass the tests was to remove the one line that messed up the timing on the race conditions (https://github.com/viamrobotics/rdk/pull/2236/commits/b08b60d077676d8e882823d305d435306cc0650a, reverted in https://github.com/viamrobotics/rdk/pull/2236/commits/2c6a32af9c1caa6042e482a8868e2f69032420df). The rest is already merged. In the future, we can make the tests more robust, and add that log line back in.